### ISSUE TITLE
Remove hidden advanced search link

### ISSUE THREAD

**Issue by ewohnlich** [plone/plone.app.layout#402]
https://github.com/plone/plone.app.layout/blob/master/plone/app/layout/viewlets/searchbox.pt#L46

It looks like this was kept when the search was refactored for Bootstrap 5, but I don't think it's needed? It goes to the same URL as the search form. In my org, an accessibility reviewer tells me this violates Criterion 2.4.3 in WCAG 2.0

**Comment by ewohnlich** [comment]
There are a couple other places that have links in a .hiddenStructure section. Again, these effect tab order and I'm not sure they serve any purpose for sight impaired users

- https://classic.demo.plone.org/en/demo/an-event - hidden ul with event attributes
- Header of the navigation portlet wherever it is present, e.g. https://classic.demo.plone.org/en/demo/an-event. This links to the site map, which makes some sense, but it is not clear why it is hidden
